components/tenant-fetcher/internal/tenant/entity.go

```diff
@@ -0,0 +1,17 @@
+package tenant
+
+type Entity struct {
+	ID             string       `db:"id"`
+	Name           string       `db:"external_name"`
+	ExternalTenant string       `db:"external_tenant"`
+	ProviderName   string       `db:"provider_name"`
+	Initialized    *bool        `db:"initialized"` // computed value
+	Status         TenantStatus `db:"status"`
+}
+
+type TenantStatus string
+
+const (
+	Active   TenantStatus = "Active"
+	Inactive TenantStatus = "Inactive"
```

Comment:
Unused Inactive status constant, do we need it?

Reply:
Removed this one and I'm now using the director's one.